src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -88,16 +95,16 @@ https://{datacenterURI}/financialintegration/fi/v4/companies/transactiontypes/{d
 
 Name|Type|Format|Description
 ---|---|---|---
-`docType`|`string`|`path`|**Required** The financial document type to return. Only one type of transaction can be retrieved at a time. Supported values are: `expense`, `invoice`, `cashadvance`.
-`offset`|`string`|`query`|Starting page offset.
+`docType`|`string`|`path`|**Required** The financial document type to return. Only one type of transaction can be retrieved at a time. Supported values are: `expense`, `invoice`, `cashadvance`, `payroll`, `obligation`.
```

Comment:
Should the docType supported values be the same for all endpoints? Right now two are showing a list of five values, and two are showing a list of three supported values.

Reply:
Changes made.  Please review.